app/submitter/converter_v2.py

```diff
@@ -113,7 +113,7 @@ def get_payload_data(
     routing_path: RoutingPath,
     metadata: MetadataProxy,
     response_metadata: Mapping,
-) -> Union[OrderedDict[str, Any], dict[str, Union[list[Any]]]]:
+) -> Union[OrderedDict[str, Any], dict[str, Union[list[Any], object]]]:
```

Comment:
Why is it returning `object`?

Reply:
Can't seem to fix the issue here at the moment, mypy seems to think the generation of answers/lists in the data payload can be returned as an object 😐

Reply:
OK, removed this now 👍